### DISCUSSION TITLE
No data in Fleet or Playbook on fresh install

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#1827] [Unsupported Versions] [answered]
I installed a manager and a few sensor nodes on our network. On install there were some errors, but they were fairly easy to fix.
However, when watching the training videos on Youtube, I saw that the Playbook interface should be populated with some plays, as well as the Fleet interface that should have at least the manager node. 
My install has neither interfaces filled with data. 
ATT&CK Navigator also appears to be empty.

Could this be an issue with the MySQL container? "docker ps" shows its status as "unhealthy", in /opt/so/log/mysql/mysqld.log, everything seems normal except the following lines:
`2020-11-02T16:09:23.106494Z 2 [Note] Got an error reading communication packets`
`2020-11-02T16:11:29.707898Z 12 [Note] Aborted connection 12 to db: 'fleet' user: 'fleetdbuser' host: '172.17.0.1' (Got an error reading communication packets)`

"docker logs so-mysql" does not show any special output.

EDIT: 
I was able to manually add my manager to Fleet by running the following command (in foreground):
`sudo /usr/local/so-launcher/bin/launcher --hostname=<redacted> --enroll_secret=<redacted>`
I assume this should be run as a service somewhere?
EDIT 2: 
I found the service (launcher.so-launcher.service), its status was failed, with errors. The errors indicate that there's an issue with the certificate being distributed to hostname, but not FQDN (which I selected for my server to listen to).
[kolide_launcher.log](https://github.com/Security-Onion-Solutions/securityonion/files/5493696/kolide_launcher.log)
EDIT 3:
Fleet issues have been resolved, clients could not connect due to certificate issue. By changing "fleet_custom_hostname" to <hostname> (not FQDN) instead of "" in /opt/so/saltstack/local/pillar/global.sls and then regenerating the installers using the following command:
`sudo salt-call state.apply fleet.event_gen-packages`

Playbook being empty comes from the unauthorized "anonymous" user trying to create the plays. In /opt/so/saltstack/local/pillar/global.sls a variable is declared for a Playbook "api_key", but has no value.

**Comment by weslambert** [comment, marked as answer]
Thanks for reporting! We are tracking all of these issues in other threads.
